src/com/tw/go/plugin/cmd/Console.java

```diff
@@ -13,8 +13,12 @@ public static CommandLine createCommand(String... args) {
         gitCmd.addArguments(args);
         return gitCmd;
     }
-
+    
     public static ConsoleResult runOrBomb(CommandLine commandLine, File workingDir, ProcessOutputStreamConsumer stdOut, ProcessOutputStreamConsumer stdErr) {
+        return runOrBomb(commandLine, workingDir, stdOut, stdErr, commandLine.toString());
```

Comment:
I saved previous behaviour in method with 4 params and added new one with 5 and pass different prettyMessage only where needed